cmd/livepeer_cli/wizard_transcoder.go

```diff
@@ -185,3 +189,58 @@ func (w *wizard) callReward() {
 	fmt.Printf("Calling reward for round %v\n", c)
 	httpGet(fmt.Sprintf("http://%v:%v/reward", w.host, w.httpPort))
 }
+
+func (w *wizard) vote() {
+	if w.offchain {
+		glog.Error("Can not vote in 'offchain' mode")
+		return
+	}
+
+	fmt.Print("Enter the contract address for the poll you want to vote in -")
+	poll := w.readStringAndValidate(func(in string) (string, error) {
+		if !ethcommon.IsHexAddress(in) {
+			return "", fmt.Errorf("invalid hex address address=%v", in)
+		}
+		return in, nil
+	})
+
+	var (
+		confirm = "n"
+		choice  = types.VoteChoice(-1)
+	)
+
+	for confirm == "n" {
+		choice = types.VoteChoice(-1)
+		w.showVoteChoices()
+
+		for {
+			fmt.Printf("Enter the ID of the choice you want to vote for -")
+			choice = types.VoteChoice(w.readInt())
+			if ok := choice.IsValid(); ok {
+				break
+			}
+			fmt.Println("Must enter a valid ID")
+		}
+
+		fmt.Printf("Are you sure you want to vote \"%v\"? (y/n) -", choice.String())
+		confirm = w.readStringYesOrNo()
+	}
+
+	data := url.Values{
+		"poll":     {poll},
+		"choiceID": {fmt.Sprintf("%v", int(choice))},
+	}
+
+	result := httpPostWithParams(fmt.Sprintf("http://%v:%v/vote", w.host, w.httpPort), data)
+
+	fmt.Printf("\n %v \n", result)
```

Comment:
Don't think we need the leading and trailing space as it will show up as:

```
 vote success
```

rather than 

```
vote success
```